index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <div class="container">
+        <div class="headers">
+
+            <h1 id="title">Survey Form</h1>
+            <p id="description">This is a survey form.</p>
+        </div>
+        <form id="survey-form">
+
+
+            <label for="name" id="name-label">Name:</label><br>
+            <input type="text" id="name" name="name" placeholder="Your Name" required><br>
+            <label for="email" id="email-label">Email:</label><br>
+            <input type="email" id="email" name="email" required placeholder="example@gmail.com"><br>
+            <label for="number" id="number-label">Age:</label><br>
+            <input type="number" id="number" name="number" min="1" max="99" placeholder="Age" required><br>
+
+            <label for="gender">Gender:</label><br>
+
+            <select name="gender" id="dropdown">
+                <option value="male">Male</option>
+                <option value="female">Female</option>
+                <option value="others">Others</option>
+
+
+            </select>
+            <br>
+            <fieldset>
+                <legend for="cars">Choose a car:</legend><br>
+                <input type="radio" id="benz" name="fav_car" value="benz">
+
+                <label for="benz">Benz</label>
+                <input type="radio" id="bmw" name="fav_car" value="bmw">
+                <label for="bmw">BMW</label>
+                <input type="radio" id="Audi" name="fav_car" value="audi">
+                <label for="audi">Audi</label>
+            </fieldset>
+            <fieldset>
+                <legend for="cars">Familiar coding languages:</legend><br>
+                <input type="checkbox" id="html" name="coding_lang" value="html">
+
+                <label for="html">HTML</label>
+                <input type="checkbox" id="css" name="coding_lang" value="css">
+                <label for="css">CSS</label>
+                <input type="checkbox" id="others" name="coding_lang" value="others">
+                <label for="others">Others</label>
+            </fieldset>
+            <label for="w3review">Additional Comments:</label><br>
+
+            <textarea id="w3review" name="w3review" rows="4" cols="50">Enter your additional comments here.
+</textarea><br>
+            <button id="submit" class="submit-button">Submit</button>
```

Comment:
I think we can make this comment better by making our comments more of a suggestion rather than an instruction. don't forget, in our previous lesson on [how to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) we were taught to give suggestions rather than orders😉 this has a way of improving your communication with the student🤗
<img width="866" alt="no_orders_cr" src="https://user-images.githubusercontent.com/64643995/213031972-24fbd011-adfd-471c-b56c-71b87537b8fa.png">

also I think we can even do more by politely explaining why we are requesting the change, this makes our comments descriptive and understandable🤗 don't forget we were also taught this in the [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)😉
<img width="581" alt="explain-crr" src="https://user-images.githubusercontent.com/64643995/213032160-e16d0286-4661-47d4-99d6-b4239d80b5c7.png">